app-tasks/dags/aoi-monitoring/find_aoi_projects_to_update.py

```diff
@@ -30,7 +30,7 @@
 dag = DAG(
     dag_id='find_aoi_projects_to_update',
     default_args=default_args,
-    schedule_interval=None,
+    schedule_interval=datetime.timedelta(minutes=5),
```

Comment:
We should bump this way up probably - more like 6-8 hours probably. We are not adding imagery more than a few times a day at this point.